pyweek36/main.py

```diff
@@ -195,9 +199,17 @@ def on_update(self, delta_time):
         # Move items in the physics engine
         self.physics_engine.step()
 
+        self.camera.move_to(
```

Comment:
https://github.com/letsbuilda/pyweek36/blob/main/pyweek36/main.py#L155-L156

Moving the camera will offset the cursor position, so you'll have to adjust for that in the bullet angle calculation too